CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
 # UNRELEASED
+<!-- Add CHANGELOG entry to this section for any PR awaiting the next release -->
+<!-- Please also include if this is a Bug Fix, Enhancement, or Feature -->
+
+## Features
+* Added support for assigning agent pools at the organization level via a `default` agent pool by @SwiftEngineer [#762](https://github.com/hashicorp/go-tfe/pull/762)
```

Comment:
This is a little incomplete! It should probably be multiple bullet points. 

- Org-level default agent pool
- Org-level default execution mode
- New `WorkspaceSettingOverwritesOptions` field for allowing workspaces to defer some settings to a default from their org or project.

Reply:
Good callout on adding `WorkspaceSettingOverwritesOptions` on there and the extra backticks!

I made one small change in that I merged the top two suggestions ("Org-level default agent pool" and "Org-level default execution mode") into one. That's because I don't want to give users the impression that they can set a default agent pool WITHOUT setting a default execution mode. In other words, the default agent pool depends on the default execution mode being set to `agent`.